agroal/src/main/resources/schema/wildfly-agroal_1_0.xsd

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I will not review the XML schema as it should be derived from the management model instead of tweaking the parser/marshaller in `AgroalObjectAttribute`.